user_research/Device/digitalData.md

```diff
@@ -0,0 +1,6 @@
+# 2012 AEC![devices](../Pictures/Devices.png)
+![ageInternet](../Pictures/ageAppropriation.png)
+![ageSmartphone](../Pictures/smartphoneAge.png)
+![mobinautes](../Pictures/mobinautes.png)
+# 2011 CREDOC
+![deviceEvolution](../Pictures/repartitionDeviceFoyerCREDOC.png)
```

Comment:
Data from CREDOC were in fact published in 2014 (June). I am sorry for the mistake.

Smartphone equipment is about 29% for CREDOC, 39% is percentage in 2013. The gap of 3% can be explained more easily =)
So you want me to send you the entires files on you mailbox or that I check samples' informations?